docs/faqs.rst

```diff
@@ -0,0 +1,76 @@
+FAQs
+====
+This page provides the frequently asked questions (FAQs) about ``teracy-dev``.
+
+Why teracy-dev? Why not just Docker?
+------------------------------------
+Docker is great to work with, it solves a lot of problems on development, distribution, and production deployment. It works great on Linux, however it's very challenging to make it work universal and consistent on Mac and Windows. There are lots of efforts to solve this problem, from Docker themselves and from Docker community, too. But, we haven't achieved that stage yet (until ``teracy-dev``). ``teracy-dev`` is an universal Productive Development Platform With Docker on macOS, Linux and Windows.  With ``teracy-dev``, anyone from any OS platforms could collaborate and help each other with no differences in the development environment, and many other reasons
+why you should use ``teracy-dev`` to develop software more easily, see details at
+http://blog.teracy.com/2016/12/20/teracy-dev-the-only-truly-universal-productive-development-platform-with-docker-on-macos-linux-and-windows/.
+
+How do I know which version of ``teracy-dev`` that I'm using?
+-------------------------------------------------------------
+Currently, to know the version of teracy-dev which you are using, you need to follow the steps: 
+First, you need to ``git branch`` to know the current branch, then ``git log`` to know the info about the latest commits. Normally, we have the commits about release the new version, for example, "Merge pull request #313 from hoatle/tasks/#312-release-v0.5.0-b3".
+
+We'll provide a simpler way soon at https://github.com/teracyhq/dev/issues/366.
+
+
+After ``vaggrant ssh``, why cannot I run the ``grunt`` or ``npm`` command in the VM?
+------------------------------------------------------------------------------------
+
+The ``grun`` or ``npm`` command is not used in the ``teracy-dev`` latest version anymore. Instead,
+you should install docker and use the ``docker-compose`` command.

```

Comment:
by default, `docker` and `docker-compose` is installed by default and you should use these two.